src/scripts/main.js

```diff
@@ -1,3 +1,36 @@
 'use strict';
 
 // write your code here
+const body = document.querySelector('body');
+
+const promise1 = new Promise((resolve, reject) => {
```

Comment:
You don't use reject here
```suggestion
const promise1 = new Promise((resolve) => {
```